devnet/substrate_bob.Dockerfile

```diff
@@ -13,17 +13,22 @@ RUN go mod download
 COPY ./devnet .
 RUN go run cmd/update-dd-agent-confd/main.go -n=${METRICS_NAMESPACE} -t=key:alice > conf.yaml
 
-FROM parity/polkadot:v0.9.17
+FROM parity/polkadot:${POLKADOT_VERSION}
 
-ARG key
-ARG CHAIN=cross-client
+ARG POLKADOT_VERSION
+# Using a genesis file with 3 authority nodes (alice, bob, charlie) generated using polkadot $POLKADOT_VERSION
+ARG CHAIN=3-auth-node-${POLKADOT_VERSION}
 ARG DD_API_KEY=somekey
+ARG key
 
 ENV DD_API_KEY=${DD_API_KEY}
 ENV CHAIN=${CHAIN}
 ENV key=${key}
 
 USER root
+RUN gpg --recv-keys --keyserver hkps://keys.mailvelope.com 9D4B2B6EB8F97156D19669A9FF0812D491B96798
+RUN gpg --export 9D4B2B6EB8F97156D19669A9FF0812D491B96798 > /usr/share/keyrings/parity.gpg
```

Comment:
I was getting problem in the `apt update` because no pub keys were provided, so I follow the Polkadot README ([link](https://github.com/paritytech/polkadot#debian-based-debian-ubuntu)) installation guide that adds these gpg keys